db/db.js

```diff
@@ -0,0 +1,20 @@
+const MySql = require('mysql');
+const Promise = require('bluebird');
+
+const connection = MySql.createConnection({
+  host: 'localhost',
+  user: 'student',
+  password: 'student'
+})
+
+const db = Promise.promisifyAll(connection, { multiArgs: true });
+
```

Comment:
i need to figure this out it seems like its keeping things alot smoother for you unlike mine (promisifyAll)